index.js

```diff
@@ -38,11 +39,11 @@ if (platform === 'win32') {
   executable = 'geckodriver.exe';
 }
 
-process.stdout.write('Downloading geckodriver... ');
+process.stdout.write(`Downloading geckodriver from ${downloadUrl}...\n`);
```

Comment:
this breaks tests, https://github.com/vladikoff/node-geckodriver/blob/master/test/index.js#L9